bundle/manifests/kuadrant.io_authpolicies.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.13.0
+    controller-gen.kubebuilder.io/version: v0.10.0
```

Comment:
You may need to `rm -rf ./bin/controller-gen`, then `make generate manifests bundle` again.

Ref.: https://github.com/Kuadrant/kuadrant-operator/pull/277